unet/test_train.py

```diff
@@ -0,0 +1,10 @@
+from model import get_unet
+from train import train_unet
+
```

Comment:
from datetime import datetime

Reply:
???

Reply:
Ah, I see what you're doing below. Make a suggestion , not a comment; or just make the commit change yourself; I'd accept it.